# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/rust_cascade/Cargo.toml
# Commit: 16350e08d34f
# Full Hash: 16350e08d34ff06c7f46745c6b160c7a7bda9140
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2019-12-06 04:43:57
# Regressor Bug: 1586855
# File Overlap Count: 6
# Description:
#   bug 1586855 - incorporate CRLite filters into cert_storage r=jcj,kjacobs
#   
#   This patch implements CRLite lookups for TLS server certificate revocation
#   information in telemetry-only mode. It adds a new preference
#   "security.pki.crlite_mode" to control the behavior of this feature. Setting
# ==============================================================================

diff -r 64fd40663930 -r 16350e08d34f third_party/rust/rust_cascade/Cargo.toml
--- a/third_party/rust/rust_cascade/Cargo.toml	Thu Dec 05 22:23:54 2019 +0000
+++ b/third_party/rust/rust_cascade/Cargo.toml	Thu Dec 05 22:41:53 2019 +0000
@@ -12,16 +12,10 @@
 
 [package]
 name = "rust_cascade"
-version = "0.3.4"
-authors = ["Mark Goodwin <mgoodwin@mozilla.com>"]
+version = "0.5.0"
+authors = ["Mark Goodwin <mgoodwin@mozilla.com>", "Dana Keeler <dkeeler@mozilla.com>"]
 description = "A simple mmh3 based bloom filter cascade implementation in Rust."
 license = "MPL-2.0"
-[dependencies.bit-vec]
-version = "0.5.1"
-
-[dependencies.bit_reverse]
-version = "0.1.7"
-
 [dependencies.byteorder]
 version = "1.3.1"
 
@@ -30,6 +24,3 @@
 
 [dependencies.murmurhash3]
 version = "0.0.5"
-
-[dependencies.rand]
-version = "0.*"